### ISSUE TITLE
Feature Request: read files from an existing workspace and request AI to make changes

### ISSUE THREAD

**Issue by NoCLin** [AntonOsika/gpt-engineer#95]
This will be very useful when we have made modifications to the workspace generated by AI.

We can then ask the AI to continue modifying the code, just like a PM.

**Comment by psirdev** [comment]
I proposed it here: https://github.com/AntonOsika/gpt-engineer/issues/79

I think I can work on this, would be awesome

**Comment by NoCLin** [comment]
> I proposed it here: #79
> 
> I think I can work on this, would be awesome

Thanks Pablo. Maybe we can do it together :)

**Comment by clickbrain** [comment]
@psirdev - I posted this in #79, but wanted to post here for @NoCLin to see as well. 

I agree with this. I've used the standard ChatGPT interface to interact and create applications, but I almost always have to go back to the prompt with errors that resulted from running the code. Having a way to get ChatGPT to add functionality or fix errors would be great.

As an example, I just had ChatGPT create a Telegram Bot, but one of the libraries it uses is throwing an error when I execute main.py. When I looked it up, it looks like it is an error caused by capitalizing the word Filters. It would be great if I could start it up again and tell gptengineer, that I ran into this error and to fix it.

In addition, I know after using an application that I will want it to add new features or make changes to functionality and beig able to have it read in or remember what it did on a project would be ideal.

Finally, I would like to be able to tell it to provide deployment instructions. This is important, because some users of this are using it because they don't know how to code nor deploy an application to a server and in my specific case, deploy a bot in telegram, etc. I put those instructions in main and it added it as part of the specification, but did not provide any instructions.
